ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_client.py

```diff
@@ -37,9 +38,14 @@ class DaprWorkflowClient:
        This client is intended to be used by workflow application, not by general purpose
        application.
     """
-    def __init__(self, host: Optional[str] = None, port: Optional[str] = None):
+    def __init__(self, host: Optional[str] = None, port: Optional[str] = None, apiToken: Optional[str] = None):
         address = getAddress(host, port)
-        self.__obj = client.TaskHubGrpcClient(host_address=address)
+        metadata = []
+        if apiToken is not None:
```

Comment:
Are you sure you even need the API token is parameter? I'd rather refer people to using the settings package to set the token:

```python
from dapr.conf import settings
settings.DAPR_API_TOKEN='mytokenhere'
```

And as you know that is only necessary if the token itself isn't already set as an environment variable.

Reply:
You can keep the parameter if you'd like to have it for consistency with other SDKs - but if other SDKs do not use this, then I recommend also removing it here.

Reply:
I have removed api_token option. It will pick only if corresponding env. setting is there.
python DaprClient also doing the same thing - so to maintain consistency.